frontend/packages/ux-editor/src/components/config/editModal/EditOptions/OptionTabs/utils/optionsUtils.test.ts

```diff
@@ -198,40 +197,47 @@ describe('optionsUtils', () => {
   describe('IsOptionsIdReferenceId', () => {
     it('should return true if options ID is a string and options ID is not from library', () => {
       const optionsId = 'another-id';
+      const optionListIds: string[] = optionListIdsMock;
```

Comment:
Now I am being extremely picky, but I actually meant to hard code the optionListIds so it is easy to see, in this test e.g., that the optionsId is NOT present in the optionListIds.